### PR TITLE
Add verification steps for CRDs.

### PR DESCRIPTION
**Pull request by andraxylia**

### CODE REVIEW

**Comment by ericvn** [comment]
Note that the correct number is now 58 per https://github.com/istio/istio.io/pull/3341 and testing on rc-0.

File: content/docs/setup/kubernetes/helm-install/index.md [hunk 1]
```diff
@@ -94,6 +94,15 @@ Choose this option if your cluster doesn't have [Tiller](https://github.com/kube
 
     {{< /text >}}
 
+1. To verify all Istio's CRDs were committed in the Kubernetes api-server, check that all CRDs
+that Istio uses are instantiated.  Verify that the number of total CRDs created was `56` for Istio:
```
Comment (ericvn) [inline content/docs/setup/kubernetes/helm-install/index.md]: Note that the correct number is now 58 per #3341 and testing on rc-0.

File: content/docs/setup/kubernetes/helm-install/index.md [hunk 2]
```diff
@@ -94,6 +94,15 @@ Choose this option if your cluster doesn't have [Tiller](https://github.com/kube
...
+
+    {{< text bash >}}
+
+    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
```
Comment (ericvn) [inline content/docs/setup/kubernetes/helm-install/index.md]: You might want to add the 58 as output in your text:

    {{< text bash >}}
    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
    58
    {{< /text >}}

Comment (andraxylia) [reply content/docs/setup/kubernetes/helm-install/index.md]: done

**Pull request merged**